clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/orchestration/AnnotationsBasedAtomicOperationsRegistry.groovy

```diff
@@ -57,27 +58,29 @@ class AnnotationsBasedAtomicOperationsRegistry extends ApplicationContextAtomicO
       value instanceof AtomicOperationConverter
     }.values().toList()
 
+    converters = converters.findAll { AtomicOperationConverter cv -> cv.acceptsVersion(version) }
+
     if (!converters) {
       throw new AtomicOperationConverterNotFoundException(
-        "No atomic operation converter found for description '${description}' and cloud provider '${cloudProvider}'"
+          "No atomic operation converter found for description '${description}' and cloud provider '${cloudProvider}' at version '${version}'"
```

Comment:
I think if you add a newline to break up a line you typically indent twice, that's what intellij did for me when I made the edit here. Does that sound right?